ale_linters/json/vscodejson.vim

```diff
@@ -1,6 +1,22 @@
 " Author: Dalius Dobravolskas <dalius.dobravolskas@gmail.com>
 " Description: VSCode json language server
 
+call ale#Set('json_vscodejson_executable', '<auto>')
+
+function! ale_linters#json#vscodejson#GetExecutable(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'json_vscodejson_executable')
+
+    if l:executable is# '<auto>'
+        if ale#engine#IsExecutable(a:buffer, 'vscode-json-languageserver')
```

Comment:
Good you put the new name here, which will become the most likely over time, as ALE will store the result of the executable check all the time if there is an executable with that name, but will only store the result of an executable check failing if `ale_cache_executable_check_failures` is set to `1`.